include/myst/gcov.h

```diff
@@ -4,9 +4,11 @@
 #ifndef _MYST_GCOV_H
 #define _MYST_GCOV_H
 
+#include <errno.h>
 #include <myst/libc.h>
 #include <stdio.h>
+#include <sys/stat.h>
 
-int gcov_init_libc(libc_t* libc, FILE* stderr_stream);
+#define MYST_GCOV_STDERR ((FILE*)0x67001b41aafb4224)
```

Comment:
Paul and I were discussing this strange constant. Can we change it to 0x2222222222222222 (so it is file descriptor 2, which is standard error)?

Reply:
Added this